addons/l10n_fr_pos_cert/models/pos.py

```diff
@@ -30,7 +30,8 @@ def _check_session_timing(self):
         self.ensure_one()
         date_today = datetime.utcnow()
         session_start = Datetime.from_string(self.start_at)
-        if not date_today - timedelta(hours=24) <= session_start:
+        if not date_today - timedelta(hours=24) <= session_start and \
+            self.config_id.module_pos_hr and not self.user_has_groups("point_of_sale.group_pos_manager"):
```

Comment:
you can also remeove the condition here, and it should be ok

Reply:
Done.
Thanks for the review.

Reply:
I've let `_check_session_timing` method in case of a custom module overriding it.
If you think it's unnecessary, all this code (https://github.com/odoo/odoo/pull/80680/files#diff-f8813dd3cd73cd717266ed6309e82fcb760496bd5ca5b47da495279ea956dc6eR26-R38) can be removed.